draft-ietf-ice-pac.xml

```diff
@@ -172,37 +179,41 @@
 
     <section title="Update to RFC 8445">
       <t>
-        In order to avoid the problem raised by this document, the ICE agent
-        needs to wait enough time to allow peer-reflexive candidates to be
-        discovered. Accordingly, when a full ICE implementation begins its
-        ICE processing, as described in <xref target="RFC8445" />, Section
-        6.1, it MUST set a timer, and its duration SHOULD be
-        equal to the agent's connectivity check transaction timeout.
+        In order to avoid the problem described in this specification, this
+        section updates <xref target="RFC8445" />. It defines a new timer, Tpac,
+        and defines new procedures associated with the timer for full ICE implementations.
+      </t>
+      <t>
+        A full ICE implementation starts timer Tpac once it has received the set of remote candidates from
+        the remote peer and has sent back its set of local candidates, as described in Section 5 of <xref target="RFC8445" 
+        pageno="false" format="default"/>.
```

Comment:
If we say it this way, then trickle ICE will also need to have its own PAC rule, right?  Instead, could we not say "once it has received the remote candidate information and sent at least one local candidate"?

Reply:
I agree the text is not correct. @juberti  has alternative text in #15, and I have commented there.